bindings/go/src/ucx/request.go

```diff
@@ -46,18 +47,42 @@ func (p *UcpRequestParams) SetCallback(cb UcpCallback) *UcpRequestParams {
 	return p
 }
 
-func packParams(params *UcpRequestParams, p *C.ucp_request_param_t, cb unsafe.Pointer) uint64 {
+func packArg(cb UcpCallback) unsafe.Pointer {
```

Comment:
`Arg` is confusing (everything is an arg).
As it has a specific type, maybe `packCallback[Handle?]`?

Reply:
We can change terminology from `arg` to `callback`.

Reply:
IMO that would be better